BlueprintUI/Sources/Layout/ConstrainedAspectRatio.swift

```diff
@@ -77,3 +77,21 @@ public struct ConstrainedAspectRatio: Element {
         }
     }
 }
+
+
+public extension ConstrainedAspectRatio {
+    /// Initializes with the given properties.
+    ///
+    /// - parameters:
+    ///   - aspectRatio: The aspect ratio that the content size should match.
+    ///   - contentMode: Whether the aspect ratio should be reached by expanding the content
+    ///     element's size to fill its parent or shrinking it to fit.
+    ///
+    func aspectRatio(
```

Comment:
Would it be a good idea to always match the Layout name? I might type `.con..` expecting to find `ConstrainedAspectRatio` if I was already familiar with the type. No strong opinion though, there's an argument to be made for keeping them succinct

Reply:
Yeah, that's fair, will change – I shortened it since an "Aspect Ratio" is already inherently constrained, but adding `constrained` to the front is nice for symmetry with the type name and `ConstrainedSize` as well.

Reply:
Actually, `constrainedTo(aspectRatio:` works nicely as well, going with that.